### PR TITLE
Positioning and Floating element (New york times page copy)

### PR DESCRIPTION
**Pull request by Tahirbhalli**
This is a clone of the New York Times web article, where we used:
1 `<aside>` HTML Element
2 `<nav>` HTML Element
3 `float: ` for CSS Style
As a part of the Odin Project.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Tahirbhalli and @adamclasic 

### You have done a really good job so far, however, there are issues you need to work on for your project to be marked completed.
- Write a more descriptive pull request [check this guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- Improve your readme file, make it better [check this guide](https://github.com/microverseinc/readme-template)
- A floating sidebar on the right is missing from your project, refer to the origin [website](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)
sidebar-missing
- In your navigation, use `nav` not `div ` and make use of `section, article, sidebar` in this project. Avoid segmenting using divs
- Validate your html code and rectify the error, [use this tool](https://validator.w3.org/#validate_by_upload)
- Configure your code with linter [check this guide](https://github.com/microverseinc/linters-config/tree/master/css)

### Make these changes and resubmit for another code review. I wish you all the best in your submission.
-If you are having challenges, reach out to me @rOluochKe

File: index.html [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css">
+        <link href="https://fonts.googleapis.com/css?family=Tinos&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="styles/style.css">
+        <link rel="stylesheet" href="styles/normalize.css">
+    </head>
+    <body>
+        <main>
+            <header>
+                <div class="header-cont">
```
Comment (MauricioRobayo) [inline index.html:17]: This could be a `nav` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+                    <div class="logo-cont">
+                        <a href="index.html"><img src="img/logo.png" alt="logo"></a>
+                    </div>
+                    <div class="right-cont-header">
+                        <button class="right-btns">SUBSCRIBE NOW</button>
+                        <BUtton class="right-btns">LOG IN</BUtton>
+                    </div>
+                </div>
+            </header>
+            <section>
```
Comment (MauricioRobayo) [inline index.html:37]: This could be a `main` tag.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi, this is coming into place, really great job!

But there are still some issues that are not solved:

- [x] Write a more descriptive pull request - you can edit the pull request description.
- [x] It should have a fixed position footer (it sticks to the bottom part of the screen).
- [ ] It uses [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the rightmost side.
---
image
---
**[OPTIONAL]**
- [ ] You can use then `nav` tag for the navigation bar inside the `header` and the `main` tag for the main section.

Status: **Request changes**.

**Comment by adamclasic** [comment]
very good review @MauricioRobayo 
we'll work on that, Thank you.

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+  padding-right: 15px;
+  font-size: 75%;
+  text-align: center;
+}
+
+/* footer end */
+
+.grid-container {
+  display: grid;
+  margin-left: 2rem;
```
Comment (MauricioRobayo) [inline styles/style.css:524]: Try setting the margin to the left and right to `auto`, for example: `margin: 0 auto`. That might solve the issue with the misalignment on wider screens. Take a look at this: https://www.w3schools.com/css/css_align.asp

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi, it is almost there, but 😣:

- [ ] Still missing the `aside` tag for Editor's picks section on the rightmost side. See the screenshot from the previous review.
- [ ] There is an issue on wider screens with the alignment of `.grid-container` at the `footer`, take a look at this screenshot (I apologize, didn't notice this before):
image

The project is looking great! Keep up the hard work 💪.

Status: **Request changes**.

**Pull request closed without merging**